hurumap/static/js/charts.js

```diff
@@ -14,6 +14,31 @@ Chart({
 function Chart(options) {
   var chart = {};
 
+    defaultColorbrewer = {
```

Comment:
`var defaultColorbrewer = ...`  or `var DEFAULT_COLORBREWER = ...` since we're treating these as constants.